lib/config/presets/internal/packages.ts

```diff
@@ -19,6 +19,8 @@ export const presets: Record<string, Preset> = {
   atlaskit: {
     description: 'All @atlaskit packages published by Atlassian.',
     matchPackageNames: ['@atlaskit/**'],
+    sourceDirectory: "design-system/{{ lookup (split packageName '/') 1 }}",
+    sourceUrl: 'https://bitbucket.org/atlassian/atlassian-frontend-mirror',
```

Comment:
I don't think we _set_ anything in any of these packages: presets. Until now they're purely "matching" rules which only work if extended

Reply:
Thanks for the feedback